wiki/pages/team.html

```diff
@@ -33,9 +33,22 @@
             <div class="w-screen h-screen fixed top-0 left-0 grid place-items-center" x-show="focused" x-cloak>
               <div
                   @click.away="focused = false" 
-                  class="p-4 lg:w-5/12 w-9/12 border-2 border-primary-dark bg-white rounded-md">
-                <h2>{{ member.name }}</h2>
-                {{ member.description }}
+                  class="p-4 lg:w-6/12 w-9/12 border-2 border-primary-dark bg-white rounded-md">
+                  <h2>{{ member.name }}</h2>
+                  <div class = "container">
+                    <div class = "container-text">
```

Comment:
Instead of using the class "container-text", you can put all the styles directly there. See [here](https://tailwindcss.com/) for documentation. 

For this, you would have something like class="pt-4".